secretmanager/snippets/regional_samples/get_regional_secret_version.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+"""
+command line application and sample code for getting metadata about a regional
+secret version, but not the secret payload.
+"""
+
+import argparse
+
+from google.cloud import secretmanager_v1
+
+
+# [START secretmanager_v1_get_regional_secret_version]
+def get_regional_secret_version(
```

Comment:
suggestion: This name is confusing, I know it follows the SDK convention, but maybe we could rename the sample to `get_regional_secret_version_metadata`? WDYT?

Reply:
It is to access the secret version and not the metadata, so i think this should be the name.
Also, This is in consistent with current samples : https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/secretmanager/snippets/access_secret_version.py, so i dont think so it should be changed.